CI/circle_parallel.sh

```diff
@@ -74,8 +74,8 @@ elif [ "$NODE_INDEX" = "3" ]; then
   [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
   #nvm install stable
   # install v16 instead of the latest stable version
-  nvm install 16
-  nvm alias default 16
+  nvm install 18
+  nvm alias default 18
```

Comment:
Increased Node version as tests were failing due to Angular refusing to compile on NodeJS versions <18. Node 18 is required for all currently supported Angular versions ([docs](https://angular.dev/reference/versions)).